pulsar-client-cpp/lib/ClientImpl.cc

```diff
@@ -222,6 +222,12 @@ void ClientImpl::subscribeAsync(const std::string& topic, const std::string& con
             lock.unlock();
             callback(ResultInvalidTopicName, Consumer());
             return;
+        } else if (conf.isReadCompacted() && (topicName->getDomain().compare("persistent") != 0 ||
```

Comment:
Is "persistent" defined as a const anywhere?

Reply:
seems no.